quarkus/extensions/kogito-quarkus-workflow-extension-common/kogito-quarkus-workflow-common-deployment/src/main/java/org/kie/kogito/quarkus/workflow/deployment/config/KogitoWorkflowBuildTimeConfig.java

```diff
@@ -18,24 +18,28 @@
  */
 package org.kie.kogito.quarkus.workflow.deployment.config;
 
-import io.quarkus.runtime.annotations.ConfigItem;
 import io.quarkus.runtime.annotations.ConfigPhase;
 import io.quarkus.runtime.annotations.ConfigRoot;
+import io.smallrye.config.ConfigMapping;
+import io.smallrye.config.WithDefault;
+import io.smallrye.config.WithName;
 
-@ConfigRoot(name = "kogito", phase = ConfigPhase.BUILD_TIME)
-public class KogitoWorkflowBuildTimeConfig {
+@ConfigRoot(phase = ConfigPhase.BUILD_TIME)
+@ConfigMapping(prefix = "kogito")
+public interface KogitoWorkflowBuildTimeConfig {
 
     /**
      * Configuration for DevServices. DevServices allows Quarkus to automatically start Data Index in dev and test mode.
      */
-    @ConfigItem
-    public KogitoDevServicesBuildTimeConfig devservices;
+    @WithName("devServices")
+    KogitoDevServicesBuildTimeConfig devservicesConfig();
```

Comment:
Looking at usages of this, in other places we tend to consider dev and services 2 words, so this would align:
```suggestion
    KogitoDevServicesBuildTimeConfig devServicesConfig();
```